x/staking/types/pool.go

```diff
@@ -16,24 +14,10 @@ const (
 	BondedPoolName    = "bonded_tokens_pool"
 )
 
-// Pool - tracking bonded and not-bonded token supply of the bond denomination
-type Pool struct {
-	NotBondedTokens sdk.Int `json:"not_bonded_tokens" yaml:"not_bonded_tokens"` // tokens which are not bonded to a validator (unbonded or unbonding)
-	BondedTokens    sdk.Int `json:"bonded_tokens" yaml:"bonded_tokens"`         // tokens which are currently bonded to a validator
-}
-
 // NewPool creates a new Pool instance used for queries
 func NewPool(notBonded, bonded sdk.Int) Pool {
 	return Pool{
 		NotBondedTokens: notBonded,
 		BondedTokens:    bonded,
 	}
 }
-
-// String returns a human readable string representation of a pool.
-func (p Pool) String() string {
-	return fmt.Sprintf(`Pool:	
-  Not Bonded Tokens:  %s	
-  Bonded Tokens:      %s`, p.NotBondedTokens,
-		p.BondedTokens)
-}
```

Comment:
We shouldn't be removing this right?

Reply:
see https://github.com/cosmos/cosmos-sdk/pull/6490#discussion_r450950979 - `yaml` is default text encoding format